### ISSUE TITLE
Reverse arrows in RTL, misalignment

### ISSUE THREAD

**Issue by yarons** [kimai/kimai#5083]
### Describe the issue

![image](https://github.com/user-attachments/assets/1f490e71-b553-4aec-91e0-bb9311651322)
The navigation arrows are facing each other.
The username on the left (not on the top bar) is too close to the colored square.

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.22.0

### How do you run Kimai?

KIMAI CLOUD

### Which PHP version are you using?

8.1

**Comment by kevinpapst** [comment]
Ok, I fixed the invalid margin and had a look if I still use margin-right or margin-left anywhere. Last point where it is used and therefor might cause an issue with RTL is the "spinner" when re-loading a table or inline widget (e.g. on the dashboard). 

Question for the calendar:

On this screen I clicked the right button:
![image](https://github.com/user-attachments/assets/cf3fe148-acfb-46a5-893b-c947f575d1cb)

And the next page was this one:
<img width="869" alt="Bildschirmfoto 2024-09-26 um 17 40 46" src="https://github.com/user-attachments/assets/22a1e760-4466-4adb-9720-e3b1f9cbc490">

Is that correct?
